influxql/engine.go

```diff
@@ -496,11 +496,19 @@ func (m *MapReduceJob) processAggregate(c *Call, reduceFunc ReduceFunc, resultVa
 		}
 	}
 
+	firstInterval := m.interval
+	if !m.stmt.IsRawQuery {
```

Comment:
How could this be anything but true?  Isn't `processAggregate()` only called for non-raw statements?